### DISCUSSION TITLE
Cannot install gcc-multilib and g++-multilib on Ubuntu 20.10 Desktop for Raspberry Pi!

### DISCUSSION THREAD

**Discussion by xuanvuong1702** [zephyrproject-rtos/zephyr#56534] [Q&A]
**Describe the bug**

I tried the installation but Ubuntu said that it could not find any package 'g++-multilib'.

```
E:Package 'gcci-multilib' has no installation candidate

E: Unable to locate package g++multilib

E: Couldn't find any package by regex 'g++multilib'
```

**To Reproduce**

1. install ubuntu 20.10 on Raspberry Pi 4
2. install sudo apt install --no-install-recommends  gcc gcc-multilib g++-multilib

**Comment by nordicjm** [comment]
The instructions are for ubuntu x86_64 i.e. on a PC, not a raspberry pi.
